client/lib/application/src/application-list/application-list.hbs

```diff
@@ -3,6 +3,19 @@
     <a href="/application/create" class="btn btn-default">
     <i class="fa fa-plus"></i> Create Application</a>
 </div>
+<form>
+    <input
+        autofocus="autofocus"
+        value="{{filterTerm}}"
+        type="search"
+        aria-label="Enter your filter term"
+        placeholder="Application" />
+    <button
+        type="submit"
+        class="btn btn-default">
+            <i class="fa fa-filter"></i> Filter
```

Comment:
Well here it is actually filtering a list of already available items. Search is looking for items and creating a list out of nowhere, not filtering, thus the difference. But search works too. I'll change it.